docroot/profiles/custom/cgov_site/modules/custom/cgov_core/config/install/metatag.metatag_defaults.global.yml

```diff
@@ -5,12 +5,13 @@ id: global
 label: Global
 tags:
   canonical_url: '[current-page:url]'
-  title: '[cgov_tokens:cgov-title] - [cgov_tokens:cgov-trans-org-name]'
+  cgdp_domain: '[cgov_tokens:cgdp-domain]'
```

Comment:
@KateMashkinaNIH please make sure to test this in Spanish

Reply:
All use cases ( including spanish) had been added to automated tests